packages/boxel/addon/components/boxel/tab-bar/index.css

```diff
@@ -1,16 +1,49 @@
 .boxel-tab-bar {
+  --boxel-tab-bar-background-color: inherit;
+  --boxel-tab-bar-border-bottom: 1px solid var(--boxel-light-500);
+  --boxel-tab-bar-font: inherit;
+  --boxel-tab-bar-font-weight-hover: 600;
+  --boxel-tab-bar-color-active: inherit;
+
   width: 100%;
   display: flex;
   flex-direction: row;
-  border-bottom: 1px solid var(--boxel-light-500);
+  border-bottom: var(--boxel-tab-bar-border-bottom);
+  background-color: var(--boxel-tab-bar-background-color);
+  font: var(--boxel-tab-bar-font);
+}
+
+.boxel-tab-bar--spread {
+  justify-content: space-around;
+}
+
+.boxel-tab-bar__item {
+  --icon-color: currentcolor;
+
+  display: flex;
+  align-items: center;
+  padding: var(--boxel-sp-xs) var(--boxel-sp-lg);
+  gap: var(--boxel-sp-xs);
+  border-bottom: 2px solid transparent;
+}
+
+.boxel-tab-bar__item--is-inactive {
+  opacity: 0.45;
+  cursor: text;
 }
 
 .boxel-tab-bar__item:hover {
```

Comment:
```suggestion
.boxel-tab-bar__item:not(.boxel-tab-bar__item--is-inactive):hover {
```

Currently this is more specific than the inactive rule above, and the cursor shown is still pointer.